sdks/java/extensions/sql/zetasql/src/main/java/org/apache/beam/sdk/extensions/sql/zetasql/translation/impl/BeamBuiltinMethods.java

```diff
@@ -70,4 +70,7 @@
 
   public static final Method DATE_METHOD =
       Types.lookupMethod(DateFunctions.class, "date", Integer.class, Integer.class, Integer.class);
+
+  public static final Method LOGICAL_AND =
```

Comment:
This isn't used anywhere, is it? If not, please remove it.

Reply:
I think this is call reflectively in `SqlOperators.java` by its name as a string, so we may not be able to remove.

Reply:
LOGICAL_AND is added in SqlOperators as an aggregate function. This version of LOGICAL_AND is a binary scalar function.

Reply:
removed, thanks!